wocky/wocky-jingle-session.c

```diff
@@ -169,6 +169,7 @@ wocky_jingle_session_defines_action (WockyJingleSession *sess,
         if (a == WOCKY_JINGLE_ACTION_TRANSPORT_ACCEPT ||
             a == WOCKY_JINGLE_ACTION_INFO )
           return TRUE;
+	__attribute__ ((fallthrough));
```

Comment:
I'm afraid this is not a portable solution.

Reply:
What would be preferred way then -Wno-implicit-fallthrough or /* FALLTHROUGH */ comment marker?

Reply:
Actually comment should be good enough - old will ignore new... we'll see. Will re-submit with comment.